reference/7.4/Microsoft.PowerShell.Core/About/about_Scopes.md

```diff
@@ -96,6 +83,34 @@ To get all the variables in the global scope, type:
 Get-Variable -Scope global
 ```
 
+A child scope doesn't inherit the variables, aliases, and functions from the
+parent scope. However, unless an item is private, the child scope can view the
+items in the parent scope. And, it can change the items by explicitly
+specifying the parent scope, but the items aren't part of the child scope.
+
+## PowerShell scopes names
+
+PowerShell defines names for some scopes to allow easier access to that scope.
+PowerShell defines the following named scopes:
+
+- **Global**: The scope that's in effect when PowerShell starts or when you
+  create a new session or runspace. Variables and functions that are present
+  when PowerShell starts, such as automatic variables and preference variables,
+  are created in the global scope. The variables, aliases, and functions in
+  your PowerShell profiles are also created in the global scope. The global
+  scope is the root parent scope in a runspace.
+- **Local**: The current scope. The local scope can be the global scope or any
+  other scope.
+- **Script**: The scope that's created while a script file runs. The commands
```

Comment:
should **private** be here? it's not really a named scope, but it sort of looks like it

Reply:
Those are scope names, where `private` is a modifier not a name.